generator.yml

```diff
@@ -63,63 +63,63 @@ targets:
+  - &arm64_cut         {config: cuttlefish_defconfig,                                                                                         << : *arm64-triple,         << : *kernel_modules}
+  - &arm64_allmod      {config: allmodconfig,                                                                                                 << : *arm64-triple,         << : *kernel_modules}
+  - &arm64_allmod_lto  {config: [allmodconfig, CONFIG_GCOV_KERNEL=n, CONFIG_KASAN=n, CONFIG_LTO_CLANG_THIN=y],                                << : *arm64-triple,         << : *kernel_modules}
+  - &arm64_allno       {config: allnoconfig,                                                                                                  << : *arm64-triple,         << : *kernel_modules}
+  - &arm64_allyes      {config: allyesconfig,                                                                                                 << : *arm64-triple,         << : *kernel_modules}
   # CONFIG_BPF_PRELOAD disabled for all cross compiled Fedora configs: https://github.com/ClangBuiltLinux/linux/issues/1433
   - &arm64_fedora      {config: [https://src.fedoraproject.org/rpms/kernel/raw/rawhide/f/kernel-aarch64-fedora.config, CONFIG_BPF_PRELOAD=n], << : *arm64-triple,         << : *kernel_modules}
   # CONFIG_DEBUG_INFO_BTF disabled for certain SUSE configs due to pahole issue: https://lore.kernel.org/r/20210506205622.3663956-1-kafai@fb.com/
-  - &arm64_suse        {config: [https://github.com/openSUSE/kernel-source/raw/master/config/arm64/default, CONFIG_DEBUG_INFO_BTF=n],      << : *arm64-triple,         << : *kernel_modules}
-  - &hexagon           {config: defconfig,                                                                                                 << : *hexagon-triple,       << : *kernel_modules}
-  - &i386              {config: defconfig,                                                                                                 << : *i386-triple,          << : *kernel_modules}
-  - &i686_fedora       {config: [https://src.fedoraproject.org/rpms/kernel/raw/rawhide/f/kernel-i686-fedora.config, CONFIG_BPF_PRELOAD=n], << : *i386-triple,          << : *kernel_modules}
-  - &i386_suse         {config: https://github.com/openSUSE/kernel-source/raw/master/config/i386/default,                                  << : *i386-triple,          << : *kernel_modules}
-  - &mips              {config: [malta_defconfig, CONFIG_BLK_DEV_INITRD=y, CONFIG_CPU_BIG_ENDIAN=y],           kernel_image: vmlinux,      << : *mips-triple,          << : *kernel_modules}
-  - &mipsel            {config: [malta_defconfig, CONFIG_BLK_DEV_INITRD=y],                                    kernel_image: vmlinux,      << : *mipsel-triple,        << : *kernel_modules}
+  - &arm64_suse        {config: [https://github.com/openSUSE/kernel-source/raw/master/config/arm64/default, CONFIG_DEBUG_INFO_BTF=n],         << : *arm64-triple,         << : *kernel_modules}
```

Comment:
rather than re-align because we have these huge URLs inline, can we add these URLs to the list of URLs at the top of this file, then reference them with a shorter identifier inline?

Reply:
If I understand you correctly?

```yaml
urls:
...
- &arm64-suse-config-url: https://github.com/openSUSE/kernel-source/raw/master/config/arm64/default
...
configs:
...
  - &arm64_suse        {config: [*arm64-suse-config-url, CONFIG_DEBUG_INFO_BTF=n], << : *arm64-triple, << : *kernel_modules}
...
```

Sure, no worries, I will do that shortly.

Reply:
Done. PTAL.